modules/ibmz-configure-nbde-with-static-ip.adoc

```diff
@@ -0,0 +1,145 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_ibm_z/installing-ibm-z.adoc
+// * installing/installing_ibm_z/installing-restricted-networks-ibm-z.adoc
+// * installing/installing_ibm_z/installing-ibm-z-kvm.adoc
+// * installing/installing_ibm_z/installing-restricted-networks-ibm-z-kvm.adoc
+
+ifeval::["{context}" == "installing-ibm-z"]
+:ibm-z:
+endif::[]
+ifeval::["{context}" == "installing-ibm-z-kvm"]
+:ibm-z-kvm:
+endif::[]
+ifeval::["{context}" == "installing-restricted-networks-ibm-z"]
+:ibm-z:
+endif::[]
+ifeval::["{context}" == "installing-restricted-networks-ibm-z-kvm"]
+:ibm-z-kvm:
+endif::[]
+
+:_content-type: PROCEDURE
+[id="configuring-nbde-static-ip-ibmz-linuxone-environment_{context}"]
+= Configuring NBDE with static IP in an {ibmzProductName} or {linuxoneProductName} environment
+
+Enabling NBDE disk encryption in an {ibmzProductName} or {linuxoneProductName} environment requires additional steps, which are described in detail in this section.
+
+.Prerequisites
+
+* You set up the External Tang Server. See link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/security_hardening/configuring-automated-unlocking-of-encrypted-volumes-using-policy-based-decryption_security-hardening#network-bound-disk-encryption_configuring-automated-unlocking-of-encrypted-volumes-using-policy-based-decryption[Network-bound disk encryption] for instructions.
+* You have installed the `butane` utility.
+* You have reviewed the instructions for how to create machine configs with Butane.
+
+.Procedure
+
+. Create Butane config files for the control plane and compute nodes.
++
+The following example Butane configuration for a control plane node creates a file named `master-storage.bu` for disk encryption:
++
+[source,yaml]
+----
+variant: openshift
+version: 4.13.0
+metadata:
+  name: master-storage
+  labels:
+    machineconfiguration.openshift.io/role: master
+storage:
+  filesystems:
+    - device: /dev/mapper/root
+      format: xfs
+      label: root
+      wipe_filesystem: true
+  luks:
+    - clevis:
```

Comment:
In the FIPS case, the `luks` entry should also include:

```
options:
  - --cipher
  - aes-cbc-essiv:sha256
```

We need this to ensure that we're using a FIPS 140-2 compatible cipher mode.  The default cipher mode is too new for 140-2 (but is supported in 140-3).  Normally Butane handles this if it sees `boot_device.luks` and also `openshift.fips: true`, but since we're not using `boot_device` we need to do this ourselves.

I'd be inclined to footnote it and say that it's only needed in FIPS mode.

Reply:
@SNiemann15 the comment from @bgilbert catched my eyes. 
in case a customer want to have fips mode & a change for LUKS is required to adjust the cipher, we might want to document that. Because encryption is kind of base for compliance rules which also includes FIPS.